src/pages/Login/index.jsx

```diff
@@ -1,30 +1,61 @@
 import style from './style.module.css'
 // React Wavify
 import Wave from 'react-wavify'
+import { useState, useRef } from 'react'
+import axios from 'axios'
 
 function Login(props) {
+  const [client, setClient] = useState({
+    gov_id: '',
+    passcode: '',
+  })
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState(false)
+
+  function fetchLogin() {
+    setLoading(true)
+    axios.post('http://localhost:4000/api/client/login', client).then((res) => {
+      setLoading(false)
+      if (!res.data) {
+        setError('error with fetch')
+      } else {
+        window.localStorage.setItem('access_token', res.data.access_token)
```

Comment:
use `react-router-dom` to move the user to the homepage once the request is done